src/test/java/unit/in/ac/bits/protocolanalyzer/utils/BeautifyTest.java

```diff
@@ -0,0 +1,45 @@
+package unit.in.ac.bits.protocolanalyzer.utils;
+
+import in.ac.bits.protocolanalyzer.utils.Beautify;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
```

Comment:
You can add the following rule to avoid the warning messages.
```java
@SuppressWarnings("PMD.AvoidUsingHardCodedIP")
@java.lang.SuppressWarnings("squid:S1313")
```

### References ###
https://stackoverflow.com/questions/10971968/turning-sonar-off-for-certain-code
https://sbforge.org/sonar/rules/show/squid:S1313?layout=false
https://stackoverflow.com/questions/39109228/how-can-we-ignore-some-sonarqube-rules-in-java
http://pmd.sourceforge.net/pmd-5.0.5/rules/java/basic.html
http://pmd.sourceforge.net/pmd-4.3/suppressing.html